templates/docs/component-status.md

```diff
@@ -29,6 +29,12 @@ When we add, make significant updates, or deprecate a component we update their
       <td>2.22.0</td>
       <td>The new <code>p-separator</code> component has been added to be used as a separator between content blocks.</td>
     </tr>
+    <tr>
```

Comment:
When deprecating and suggesting new component, we should add info about Code snippet being added as well.